src/lib/Components/Inbox/Conversations/Preview/InvoicePreview.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react"
+import { Image, Text, TouchableHighlight } from "react-native"
+import * as Relay from "react-relay"
+import styled from "styled-components/native"
+
+import colors from "../../../../../data/colors"
+import fonts from "../../../../../data/fonts"
+
+import AttachmentPreview, { AttachmentProps } from "../Preview/Attachment/AttachmentPreview"
+
+const Container = styled.View`
+  border-width: 1;
+  border-color: ${colors["gray-regular"]};
+  flex: 1;
+  flex-direction: row;
+`
+
+const TextContainer = styled.View`
+  flex: 1;
+  flex-direction: column;
+  margin-left: 25;
+  align-self: center;
+`
+
+const Icon = styled(Image)`
+  resize-mode: contain;
+  width: 40;
+  margin-top: 12;
+  margin-left: 12;
+  margin-bottom: 12;
+`
+
+interface Props extends AttachmentProps, RelayProps {}
+
+export const InvoicePreview: React.SFC<Props> = ({ invoice, onSelected }) =>
+  <AttachmentPreview attachment={invoice as any} onSelected={onSelected}>
+    <Container>
+      <Icon source={require("../../../../../../images/pdf.png")} />
+      <TextContainer>
+        <Text>
+          Payment Request{"\n"}
+          {invoice.total}
```

Comment:
I have to still add in the varying 'state' stylings: green for paid, the 'pay' button for unpaid, an grey for the rest.